gax/src/main/java/com/google/api/gax/batching/FlowController.java

```diff
@@ -110,8 +110,31 @@ public String toString() {
    */
   @BetaApi
   public enum LimitExceededBehavior {
+    /**
+     * Throws {@link MaxOutstandingElementCountReachedException} or {@link
+     * MaxOutstandingRequestBytesReachedException}.
+     *
+     * <p>This might be appropriate in interactive scenarios. For example, a web server might catch
+     * these exceptions and report to the user that the system is overloaded and that the user could
+     * try again later.
+     */
```

Comment:
Might this also be useful if some wants to add rate limiting logic?

Reply:
Done